### ISSUE TITLE
Autocomplete color + start page

### ISSUE THREAD

**Issue by Keplyx** [Timocop/BasicPawn#8]
Back to complain even more! ;)

I'm using a custom theme and the inverted colors, but the autocomplete window (the one in the bottom-right corner) is impossible to read:

![spectacle h14721](https://cloud.githubusercontent.com/assets/23726131/26540629/8d336cee-4452-11e7-9b43-8cf3447c4424.png)

Would it be possible to change the font color to white for this window? I know I can use the popup one to read, but sometimes I need to scroll down the autocomplete.

Also, the start page is very useful, but even if I select which files I want to open, an unnamed file is created:
![spectacle d14900](https://cloud.githubusercontent.com/assets/23726131/26540693/dfe3cbc8-4452-11e7-8200-eb6203b7135b.png)

Would it be possible to only open the selected files with the start page?

By the way, I'm using wine 2.9 now (updates are coming super fast)

**Comment by Timocop** [comment]
Seems to be a typical wine issue. When Textboxes are _ReadOnly_ the text color is still black no matter what. I'll look into it!

**Comment by Timocop** [comment]
Both fixed. The RichTextBox had some coloring problems on Wine when text changes, found an easy fix for that. You can download the latest dev build from [here.](https://ci.appveyor.com/project/Timocop/basicpawn/build/artifacts)
Keep dem bugs and ideas comin'! :P

**Comment by Keplyx** [comment]
Thanks, will do!

**Comment by Keplyx** [comment]
well the dev version has the auto-completion problem back ;)
I'm going to stick to the 0.715 for now

**Comment by Timocop** [comment]
Whaa... How? The 0.715 and the Dev version are pretty much identical. [Try this one.](https://ci.appveyor.com/project/Timocop/basicpawn/build/artifacts) Switched to VS2017.

**Comment by Keplyx** [comment]
The latest dev works, but I tried again the first dev and it also works now :/

May be related to the way I updated the software.
At first, I just moved the dev files over the 0.715 ones, to keep my config and stuff.
But this time, I created a new folder, moved the dev files inside it, and added my config after.

I don't really know why it made a difference, but hey, it works now, sorry for the false alert ;)
